.gitignore

```diff
@@ -1 +1,2 @@
 target/
+/APIKey.txt
```

Comment:
I dont think you need the leading / here. It might actually cause problems. if APIKey.txt is in the same level as .gitignore, we can remove the leading /